src/main/resources/docs/description/unused.md

```diff
@@ -1,15 +1,19 @@
+# unused
 
-Detects when variables are declared and never used. For example, variables `b` and `c` can be removed in this example:
-
+This option warns when you define and never use your variables. It is very
+useful for general code cleanup, especially when used in addition to
+`undef`.
+    // jshint unused:true
     function test(a, b) {
       var c, d = 2;
-
       return a + d;
     }
-
     test(1, 2);
-
-In addition to that, this pattern will also warn you about unused global variables declared via /*global ... */ directive.
-
-[Source](http://www.jshint.com/docs/options/#unused)
```

Comment:
Why are we removing the source from the explanation files?

Reply:
No particular reason. Should I re-add it?

Reply:
Well, you tell me :D Do we still have this standard? Do we have in the other tools? Do we prefer to have this, or is there a plan to remove them? :p 

I would not remove them in this scope unless there is a previous agreement or an argument to drop them, your call!